src/Workspaces/CSharpTest/Formatting/FormattingTests.cs

```diff
@@ -10978,5 +10979,34 @@ public async Task TestNormalizeUsingAlias(string text, string expected)
         {
             await AssertFormatAsync(expected, text);
         }
+
+        [Fact]
+        public async Task TestExtension1()
+        {
+            await AssertFormatAsync(
```

Comment:
>AssertFormatAsync [](http://example.com/codeflow?start=18&length=17)

Am I crazy, or do some of our test methods take in the expected/code in a different order? I swear one of the reviews from earlier today had them in a different order.